SPECS/xorg-x11-font-utils/xorg-x11-font-utils.spec

```diff
@@ -35,7 +33,8 @@ BuildRequires:  pkgconfig(xorg-macros) >= 1.8
 
 Provides:       bdftopcf = %{bdftopcf}
 Provides:       fonttosfnt = %{fonttosfnt}
-Provides:       mkfontdir = %{mkfontdir}
+# mkfontdir used to be a separate package, it's now in mkfontscale            
```

Comment:
Thanks for the comment!

Reply:
mkfontdir is not a separate package any more since 1.0.7, and moved to mkfontscale 1.2.0. We upgrade mkfontscale to 1.2.2 which has the updated mkfontdir already.